content/terms/reference.en.md

```diff
@@ -1,86 +1,15 @@
 ---
-title: "Contributing terms"
+title: "Terms reference"
```

Comment:
I'd like to have the title in the navigation menu be only “Reference” since it's in the “Terms” section already, but that the HTML title is “Terms reference” so that it is not ambiguous in a history entry or tab name.